### DISCUSSION TITLE
VoiceChannel topic edit?

### DISCUSSION THREAD

**Discussion by moa12gg** [discordjs/discord.js#10170] [Q&A] [answered]
How do I change the topic of the voice channel? (v13)

I tried the following:
`client.channels.cache.get('11598340623541xxxx').edit({topic:'test1'}).then(console.log).catch(console.error);`
But this doesn't work.
-------DiscordAPIError: Invalid Form Body
-------topic: Channel topic contains at least one word that is not allowed.

another attempt:
``client.channels.cache.get('11598340623541xxxx').setToic("test").then(console.log).catch(console.error);``
But this doesn't work.
------- TypeError: client.channels.cache.get(...).setTopic is not a function

Is there a way?
Sorry I used Google Translator.

**Comment by Jiralite** [comment, marked as answer]
See #10169.
